streams/src/test/java/org/apache/kafka/streams/integration/RegexSourceIntegrationTest.java

```diff
@@ -249,7 +249,7 @@ public void testShouldReadFromRegexAndNamedTopics() throws Exception {
         assertThat(actualValues, equalTo(expectedReceivedValues));
     }
 
-    //TODO should be updated to expected = TopologyBuilderException after KAFKA-3708
+    // todo should be updated to expected = TopologyBuilderException after KAFKA-3708
```

Comment:
I'm actually more considering about the comment in line 264-267, and probably line 276 for de-capitalizing. As for `TODO` it is usually treated as a marker in most IDEs for easy finding and we'd keep it that way. For example:

https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java#L972